src/modules/MultipleGallery.php

```diff
@@ -0,0 +1,35 @@
+<?php
+/**
+ * MulitpleGallery
+ */
+namespace WPML\PB\Elementor\Modules;
+
+class MulitpleGallery extends \WPML_Elementor_Module_With_Items {
+
+    protected function get_title( $field )
```

Comment:
@GeorgeBotsev, the opening bracket `{` should also be on the same line.

Reply:
Uploading a fixed version now.